### ISSUE TITLE
Optimization - Minor - CreatedAt SQL INSERT

### ISSUE THREAD

**Issue by Heavybob** [AR-Development/PersistentEmpires-OpenSourced#4]
This is just a minor observation. 

https://github.com/AR-Development/PersistentEmpires-OpenSourced/blob/c1e4c936de5f153c5a137f94610b773239b01bb9/PersistentEmpiresServer/PersistentEmpiresSave/Database/Repositories/DBLogRepository.cs#L26

When using INSERT it appears we're also inserting a value for the CreatedAt for logs. This isn't really a big deal but it might be beneficial to instead remove this value from the insert and just alter the tables column to have the CreatedAt value. 

This would mean the server doesn't need to spend those extra precious micro seconds of time per request to get the time and instead pass that to the mysql / mariadb server.

**Comment by Snah79** [comment]
Yes thats good habit to use Now(). I do it in my ubmode like this:
var insertsql = $"INSERT INTO logs (CreatedAt, IssuerPlayerId, IssuerPlayerName, ActionType, IssuerCoordinates, LogMessage, AffectedPlayers) " +
    $"VALUES(Now(), ......

**Comment by ianespana** [comment]
> Make sure your db and server runs on same machine or at least at same timezone [...]. If you cant guarantee that, then its better to pass servers Now to sql

For future reference, this is mostly a non-consent if you design your database correctly. All you have to do is store the data as a timestamp in UTC format
